web/src/views/authentication/RegisterView.vue

```diff
@@ -0,0 +1,203 @@
+<script setup>
+import { reactive, ref } from "vue";
+
+import { registerNewUser } from "@/adapters/authentication.js";
+import PasswordField from "@/components/PasswordField.vue";
+
+function initialFormState() {
+  return {
+    firstname: "",
+    lastname: "",
+    email: "",
+    password: "",
+    birthday: "",
+  };
+}
+
+const form = reactive(initialFormState());
+const isSubmitting = ref(false);
+const successMessage = ref("");
+const errorMessage = ref("");
+
+function resetFeedback() {
+  successMessage.value = "";
+  errorMessage.value = "";
+}
+
+function resetForm() {
+  Object.assign(form, initialFormState());
+}
+
+async function handleSubmit() {
+  resetFeedback();
+  isSubmitting.value = true;
+
+  const result = await registerNewUser({ ...form });
+
+  if (result.success) {
+    successMessage.value = "Un e‑mail a été envoyé pour activer votre compte.";
+    resetForm();
+  } else {
+    errorMessage.value =
+      result.message ?? "Une erreur est survenue lors de l'inscription. Veuillez réessayer.";
+  }
+
+  isSubmitting.value = false;
+}
+</script>
+
+<template>
+  <section class="register-view">
+    <h1>Créer un compte</h1>
+
+    <form
+      v-if="!successMessage"
+      class="register-form"
+      @submit.prevent="handleSubmit"
+    >
+      <div class="form-control">
+        <label for="firstname">Prénom</label>
+        <input
+          id="firstname"
+          v-model="form.firstname"
+          type="text"
+          name="firstname"
+          autocomplete="given-name"
+          required
+        >
+      </div>
+
+      <div class="form-control">
+        <label for="lastname">Nom de famille</label>
+        <input
+          id="lastname"
+          v-model="form.lastname"
+          type="text"
+          name="lastname"
+          autocomplete="family-name"
+          required
+        >
+      </div>
+
+      <div class="form-control">
+        <label for="email">Adresse e‑mail</label>
+        <input
+          id="email"
+          v-model="form.email"
+          type="email"
+          name="email"
+          autocomplete="email"
+          required
+        >
+      </div>
+
+      <PasswordField
+        id="password"
+        v-model="form.password"
+        name="password"
+        label="Mot de passe"
+        autocomplete="new-password"
+        :required="true"
+      />
+
+      <div class="form-control">
+        <label for="birthday">Date de naissance</label>
+        <input
+          id="birthday"
+          v-model="form.birthday"
+          type="date"
+          name="birthday"
+          required
+        >
+      </div>
+
+      <button
+        type="submit"
+        :disabled="isSubmitting"
+      >
+        {{ isSubmitting ? "Création du compte..." : "Créer un compte" }}
+      </button>
+    </form>
+
+    <p
+      v-if="successMessage"
+      class="feedback success"
+      role="status"
```

Comment:
Use alert role for screen readers.
```suggestion
      role="alert"
```